swiftpkg/internal/generate_modulemap.bzl

```diff
@@ -75,6 +98,14 @@ generate_modulemap = rule(
         "module_name": attr.string(
             doc = "The name of the module.",
         ),
+        "noop": attr.bool(
+            doc = """\
+Designates whether a modulemap should be generated. If `True`, a modulemap is \
+generated. If `False`, a modulemap file is not generated and the returned \
+providers are empty.\
+""",
+            default = False,
```

Comment:
Should this doc be flipped? if `noop == true` dont we return empty?

Reply:
Yes. Good catch! I just updated the doc.